Source/Core/VideoBackends/OGL/FramebufferManager.cpp

```diff
@@ -99,8 +99,8 @@ FramebufferManager::FramebufferManager(int targetWidth, int targetHeight, int ms
 
     glBindTexture(m_textureType, m_efbDepth);
     glTexParameteri(m_textureType, GL_TEXTURE_MAX_LEVEL, 0);
-    glTexImage3D(m_textureType, 0, GL_DEPTH_COMPONENT32F, m_targetWidth, m_targetHeight,
```

Comment:
We don't want to use a depth-stencil format for GPUs that support atomic-based bounding box, as it's just wasting memory. Also, it's probably a good idea to confirm that D32F_S8 is a supported texture format, I'm not sure if it's mandated by the GL spec that all hardware supports it.